packages/deploy/src/api/platform.ts

```diff
@@ -0,0 +1,15 @@
+import { BaseApiClient } from 'defender-base-client';
+
+export class PlatformApiClient extends BaseApiClient {
+  protected getPoolId(): string {
+    return process.env.PLATFORM_DEPLOYMENT_POOL_ID || 'us-west-2_94f3puJWv';
```

Comment:
mentioned this in other PR, but I would advocate simply using `PLATFORM_POOL_ID`, etc. - here were my thoughts on it: https://github.com/OpenZeppelin/defender/pull/3507?notification_referrer_id=NT_kwDOBYQPULM1MzA0NDgxMzU2OjkyNTQwNzUy#discussion_r1068104116